hachoir/parser/guess.py

```diff
@@ -127,7 +127,8 @@ def createParser(filename, real_filename=None, tags=None):
     Create a parser from a file or returns None on error.
 
     Options:
-    - filename (unicode): Input file name ;
+    - file (unicode|io.IOBase): Input file name or
```

Comment:
"unicode" doesn't exist anymore in Python 3. The doc is outdated, can you replaced replace it with "str".